app/src/main/res/values-de-rDE/strings.xml

```diff
@@ -82,10 +82,10 @@
     <string name="action_search">Suche</string>
     <string name="action_starred">Wichtig</string>
     <string name="action_remove">Entfernen</string>
-    <string name="action_overwrite">Überschreibem</string>
-    <string name="overwrite_message">Datei mit dem Namen existiert, wird nun überschrieben</string>
+    <string name="action_overwrite">Überschreiben</string>
+    <string name="overwrite_message">Eine Datei mit dem Namen existiert bereits,sie wird nun überschrieben</string>
```

Comment:
Here there should be a space after the comma?

Reply:
ops, done now